tools/test/toolchains/api_test.py

```diff
@@ -165,13 +165,21 @@ def test_gcc_version_check(_run_cmd):
     toolchain.version_check()
     assert len(notifier.messages) == 1
     _run_cmd.return_value = ("""
-    arm-none-eabi-gcc (Arch Repository)
+    arm-none-eabi-gcc (Arch Repository) 9.1.0
```

Comment:
This appears to be checking for version 9 rather than 8.

Reply:
That's a test of the version check - checking that 9.1.0 generates a diagnostic.

Reply:
Ok, thanks.